package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "nuxt-app",
+  "private": true,
+  "type": "module",
+  "scripts": {
+    "build": "nuxt build",
+    "dev": "nuxt dev",
+    "generate": "nuxt generate",
+    "preview": "nuxt preview",
+    "postinstall": "nuxt prepare"
+  },
+  "devDependencies": {
+    "@nuxt/devtools": "latest",
+    "autoprefixer": "^10.4.16",
+    "daisyui": "^3.9.3",
+    "nuxt": "^3.7.4",
+    "postcss": "^8.4.31",
+    "tailwindcss": "^3.3.3",
+    "vue": "^3.3.4",
+    "vue-router": "^4.2.5"
+  },
+  "dependencies": {
+    "@firebase/app-types": "^0.9.0",
+    "@pinia/nuxt": "^0.5.1",
+    "firebase": "^10.5.0",
+    "firebase-admin": "^11.11.0",
+    "firebase-functions": "^4.4.1",
+    "pinia": "^2.1.7"
+  },
+  "overrides": {
```

Comment:
@Nikoszym ta sekcja "overrides" jest tutaj potrzebna? Bo powoduje u mnie błąd przy instalce:
```
npm ERR! code EOVERRIDE
npm ERR! Override for vue@^3.3.4 conflicts with direct dependency
```
Natomiast jak ją usunęłam, to zainstalowało się bez problemu.

Reply:
Overrides trzeba usunąć tylko do instalacji, bo później Pinia może źle działać